app/controllers/integrated/childcare_expenses_controller.rb

```diff
@@ -1,11 +1,14 @@
 module Integrated
   class ChildcareExpensesController < FormsController
-    def self.skip?(application)
-      application.single_member_household?
-    end
-
     include SingleExpense
 
+    def self.skip_rule_sets(application)
+      [
+        SkipRules.must_be_applying_for_food_assistance(application),
+        SkipRules.multi_member_only(application),
```

Comment:
this one seems like it should be using the shovel since `multi_member_only` would already be added by `SingleExpense`

Reply:
I had to change `SingleExpense` to no longer always include the `multi_member_only` restriction. There are controllers that use it that should also show for single members.